src/campsite-info/components/camp-rating/camp-rating.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Rating } from '@mui/material';
+import StarIcon from '@mui/icons-material/Star'
+import { ratingStyles, tooltipStyles } from './camp-rating.styles';
+
+const tooltipLabels: { [index: string]: string} = {
+  1: "Terrible",
+  2: "Bad",
+  3: "Average",
+  4: "Good",
+  5: "Awesome!",
+};
+
+function getLabelText(value: number) {
+  return `${value} Star${value !== 1 ? 's' : ''}, ${tooltipLabels[value]}`;
+}
+
+export const StarRating = () => {
+  const [value, setValue] = React.useState<number | null>(0); //this line shows the default rating
```

Comment:
If you're instantiating this state to 0, can it ever be null?

Reply:
In fact, if this is just going to be type number you can omit the explicit typing completely. TS can infer the type from your initial value.

Reply:
actually, I'm wrong here. It looks like the component typing requires the `null` option. Ignore me!